README.md

```diff
@@ -32,22 +32,54 @@ $ cd $GOPATH/src/github.com/terraform-providers/terraform-provider-vcd
 $ make build
 ```
 
+Building The Provider (the modules way)
+--------------------------------------
 
-Developing the Provider
----------------------------
-
-If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (version 1.10+ is *required*). You'll also need to correctly setup a [GOPATH](http://golang.org/doc/code.html#GOPATH), as well as adding `$GOPATH/bin` to your `$PATH`.
+Starting with version 2.1 provider started using [Go modules](https://github.com/golang/go/wiki/Modules)
+This means that it is no longer necessary to be in GOPATH.
+[See more](https://github.com/golang/go/wiki/Modules#how-to-use-modules) on how to use modules
+and toggle between modes.
 
-To compile the provider, run `make build`. This will build the provider and put the provider binary in the `$GOPATH/bin` directory.
-
-```sh
+```
+$ cd ~/mydir
+$ clone git@github.com:terraform-providers/terraform-provider-vcd
```

Comment:
For copy + paste reasons you should write `git clone git@github.com:terraform-providers/terraform-provider-vcd`